### PR TITLE
Chapter 12: Password reset

### PR DESCRIPTION
**Pull request by rashidfirhat98**

### CODE REVIEW

**Comment by rashidfirhat98** [comment]
ready

File: app/controllers/password_resets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      flash[:success] = "Password has been reset."
+      redirect_to @user
+    else
+      flash[:danger] = "Password unchanged"
+      render :edit
+    end
+  end
+
+  private
+  def get_user
```
Comment (saobv-2223) [inline app/controllers/password_resets_controller.rb]:
```suggestion
  private
  
  def get_user
```

**Review by saobv-2223 — COMMENTED** [review]
fix comment and merge

File: app/controllers/password_resets_controller.rb (hunk 1 above)
Comment (rashidfirhat98) [reply app/controllers/password_resets_controller.rb]: done

**Pull request merged**